arches/app/utils/betterJSONSerializer.py

```diff
@@ -79,7 +79,9 @@ def serialize(self, obj, **options):
         options.pop("fields", None)
         options.pop("exclude", None)
         options.pop("force_recalculation", False)
-        return json.dumps(obj, cls=DjangoJSONEncoder, sort_keys=sort_keys, **options.copy())
+        result = json.dumps(obj, cls=DjangoJSONEncoder, sort_keys=sort_keys, **options.copy())
+
+        return result.encode("utf-8") if hasattr(self, "utf_encode") else result
```

Comment:
Right now if someone sets `self.utf_encode = False` the result will still be utf encoded.  
I think what we should do is add the attribute to the __init__ method and set it to False

```
def __init__(self, **options):
    self._options = options
    self.utf_encode = False
```

Then here we can check for the value of the attribute.

Reply:
Thanks @apeters - you are right. I was just checking for the existence of `utf_encode`, but as I was setting the value to `True` it would make sense that someone would set the value to `False` and expect the result to not be utf8 encoded. I will make the change, as suggested.